RealmSwift/Object.swift

```diff
@@ -133,14 +133,16 @@ open class Object: RLMObjectBase, ThreadConfined, RealmCollectionValue {
 
     /**
      WARNING: This is an internal helper method not intended for public use.
+     It should never be overriden and is not considered part of the public API.
      :nodoc:
      */
-    public override final class func className() -> String {
+    open override class func className() -> String {
```

Comment:
Is there any reason for this override to still exist at all? I think it was just here to mark it as final, so if that doesn't actually work it can probably just be deleted.

Reply:
Removing the override fixes the issue with the tests not working properly with 8.3.3. I think the pre-3.0.0 code was the useless instance method, so like you said the right thing to do was to remove the method definition altogether.